src/sass/bootstrap-asu-extends.scss

```diff
@@ -30,7 +30,7 @@
 @import 'extends/video';
 @import 'extends/card-and-image';
 @import 'extends/grid-links';
-@import 'extends/back-to-top-buttton';
+//@import 'extends/back-to-top-buttton';https://asudev.jira.com/browse/UDS-646
```

Comment:
This component was removed from Storybook as "not yet ready for prime time." The discussion within centered around the user experience of the button and what it was really attempting to accomplish. It also had significant accessibility issues (not reachable via the keyboard when it appeared on the screen.) 

Once these changes are merged, we will need to make a corresponding change to remove the JS that fires to trigger the Back to Top button from our theme.

Reply:
This makes sense. I see you've already create the PR to remove it ;)